src/App.jsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import About from './Pages/About/About';
+import Blog from './Pages/Blog/Blog';
+import Contact from './Pages/Contact/Contact';
+import Gallery from './Pages/Gallery/Gallery';
 import Home from './Pages/Home/Home';
+import Package from './Pages/Package/Package';
 
 function App() {
   return (
-    <div className="App">
-      <Home />
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route index path="/" element={<Home />} />
+        <Route path="/about" element={<About />} />
+        <Route path="/tourpackage" element={<Package />} />
+        <Route path="/gallery" element={<Gallery />} />
+        <Route path="/contact" element={<Contact />} />
+        <Route path="*" element={<Blog />} />
```

Comment:
The component here should not be blog. It should be a generic fallback component. So rename it to something more meaningful